nuxeo-core/nuxeo-core-event/src/test/java/org/nuxeo/ecm/core/work/AbstractWorkManagerTest.java

```diff
@@ -232,7 +237,7 @@ public void testWorkManagerConfig() throws Exception {
     }
 
     @Test
-    public void testWorkManagerWork() throws Exception {
+    public void testWorkManagerWork() throws InterruptedException {
```

Comment:
For tests I don't see what this brings, I must say I'm not a fan of having detailed exceptions here.

Reply:
I know but what I mean is that it's the Sonar warnings that are the problem, not the code.

Reply:
OK, so maybe we an adjust the rules...